watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination_ui.py

```diff
@@ -163,18 +163,18 @@ def export_variables(flowsheet=None, exports=None):
         is_output=False,
     )
 
-    # Unit model, secondary WWTP
-    exports.add(
-        obj=fs.pretreatment.wwtp.energy_electric_flow_vol_inlet,
-        name="Specific energy consumption per inlet flow rate",
-        ui_units=pyunits.kWh / pyunits.m**3,
-        display_units="kWh/m3",
-        rounding=2,
-        description="Electrical energy consumption with respect to influent volumetric flow rate",
-        is_input=True,
-        input_category="Secondary Wastewater Treatment",
-        is_output=False,
-    )
+    # Unit model, secondary WWTP - uncomment if we want to consider pretreatment
+    # exports.add(
+    #     obj=fs.pretreatment.wwtp.energy_electric_flow_vol_inlet,
+    #     name="Specific energy consumption per inlet flow rate",
+    #     ui_units=pyunits.kWh / pyunits.m**3,
+    #     display_units="kWh/m3",
+    #     rounding=2,
+    #     description="Electrical energy consumption with respect to influent volumetric flow rate",
+    #     is_input=True,
+    #     input_category="Secondary Wastewater Treatment",
+    #     is_output=False,
+    # )
```

Comment:
Why not add a check like `if hasattr(fs, 'pretreatment')`, run this code that you currently have commented out. Else, skip it.

Reply:
@dangunter @MichaelPesce curious about your thoughts on implementing something like this bit as an example when considering config options for the UI